proteomic_aggregation/python/proteomic_aggregation.ipynb

```diff
@@ -0,0 +1,3591 @@
+{
```

Comment:
<p>output is json format, not yaml</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>